libs/sdk-embedding/src/internal/filterValuesResolver.ts

```diff
@@ -16,11 +24,37 @@ export function resolveFilterValues(
     return new Promise((resolve) =>
         resolve(
             filters.reduce((result, filter) => {
-                // eslint-disable-next-line no-console
-                console.log(filter);
-                // TODO switch per filter type
+                const ref = filterObjRef(filter);
+                invariant(ref, `filter without reference not supported: ${ref}`);
```

Comment:
Detail: this error message will not be very useful - the `ref` shown in it will bu null or undefined. Should we serialize the `filter` object instead?